test/recipes/70-test_key_share.t

```diff
@@ -140,7 +140,7 @@ ok(TLSProxy::Message->success(), "Non preferred key_share");
 $proxy->filter(\&modify_key_shares_filter);
 
 SKIP: {
-    skip "No ec support in this OpenSSL build", 1 if disabled("ec");
+    skip "No ec support in this OpenSSL build", 1 if disabled("ec") || disabled("ecx");
```

Comment:
Does it really fail otherwise?

Reply:
yep, the key_exchange data of this case is X25519,
I changed suppgroup and key_exchange data to P-256 when ecx disabled, please check L365 L341 in this file.